pkg/virt-launcher/virtwrap/api/schema_test.go

```diff
@@ -61,6 +61,10 @@ var exampleXML = `<domain type="kvm" xmlns:qemu="http://libvirt.org/schemas/doma
     <watchdog model="i6300esb" action="poweroff">
       <alias name="mywatchdog"></alias>
     </watchdog>
+    <rng model="virtio">
+      <rate period="2000" bytes="1234"></rate>
+      <backend model="random">/dev/random</backend>
```

Comment:
why using /dev/random when the recommended one is urandom?

Reply:
Only tests, anything would do really. Even `North Pole` would be a great choice here.

Reply:
I think it's best to use a relevant example

Reply:
Indeed you are right. Fixed that.